### PR TITLE
60 auth token handling

### LINKED ISSUE #60: Auth Token Handling
Issue that tries to address the problem of user having to copy and paste their auth token when they need to add a server.

### PR DESCRIPTION
**Pull request by samarth9201**

### CODE REVIEW

**Comment by samarth9201** [comment]
Python test test_add_server_redirect_url needs to be updated.

File: feed_amalgamator/helpers/mastodon_oauth_interface.py [hunk 1]
```diff
@@ -200,10 +200,10 @@ def add_domain_to_database(self, domain_name):
...
         # TODO: Will need to refactor the code below into several functions
         payload = {
-            'client_name': 'Test Application',
-            'redirect_uris': 'urn:ietf:wg:oauth:2.0:oob',
-            'scopes': 'read write push',
-            'website': 'https://myapp.example'
+            "client_name": "Feed Amalgamator",
+            "redirect_uris": self.REDIRECT_URI,
+            "scopes": "read write push",
+            "website": "http://127.0.0.1:5000",
```
Comment (srinivasraman18) [inline feed_amalgamator/helpers/mastodon_oauth_interface.py:206]: @samarth9201 can you please explain what this payload is used for and what the parameter "website" refers to?

Comment (samarth9201) [reply feed_amalgamator/helpers/mastodon_oauth_interface.py:206]: "website" is a required parameter for the API call. It is not used after registering the application. The most important fields in this payload is "redirect_uris" and "scopes".

**Review by srinivasraman18 — APPROVED** [review]
Tested and works.

**Pull request merged**